Initium-Odp/src/com/universeprojects/miniup/server/ODPDBAccess.java

```diff
@@ -1627,6 +1627,18 @@ public boolean awardBuff_Berry(CachedDatastoreService ds, CachedEntity character
 		}
 		return buffApplied;
 	}
+	
+	public boolean awardBuff_Elixir(CachedDatastoreService ds, CachedEntity character)
+	{
+		boolean buffApplied = false;
+		CachedEntity buff = awardBuff(ds, character.getKey(), "images/small/pixel_art_food_strange_elixir.png","Strange Elixir",
```

Comment:
This doesn't look right at all, are you sure its not small2? What about any capital letters?